README.md

```diff
@@ -8,12 +8,26 @@ The protocol is meant to be RESTful, over HTTP.
 
 The specification is currently written in **OpenAPI 3.0** and can be pretty-visualized in the [github pages][tspGhPages].
 
-## HOW-TO
+## Current version
+
+The current version of specification is currently implemented and supportd in the [Trace Compass trace-server][tcServer] (reference implementation) and what is currently supported by the [tsp-typescript-client][tspClient].
+
+Swagger can be used to generate the API version implemented in Trace Compass trace-server (see [here](#gernerate-the-specification)).
+
+## Future version
+
+Some proposal for additional endpoints and features are documented in the `./API-proposed.yaml`. All the proposed changes are still not confirmed and can change. The pretty-visualized file can be found [here][apiProposed]. A diff of the current version and future version will show the differences.
+
+Once a update has been approved it will be migrated to the main `./API.yaml` file.
+
+## Update manually
 
 The specification should be edited with the [OpenAPI (Swagger) Editor extension][vscodeOpenapi] for VS Code.
 
 The latter extension is assumed for consistent formatting of the `./API.yaml` file over time.
```

Comment:
I'd rather not open the door to manually updating API.yaml, which is now generated from swagger-core. Rename here to ./API-proposed.yaml?